pkg/controller/vsphere/reconciler_test.go

```diff
@@ -952,14 +959,16 @@ func TestDelete(t *testing.T) {
 	model, _, server := initSimulator(t)
 	defer model.Remove()
 	defer server.Close()
-	credentialsSecretUsername := fmt.Sprintf("%s.username", server.URL.Host)
-	credentialsSecretPassword := fmt.Sprintf("%s.password", server.URL.Host)
+	url := strings.Split(server.URL.Host, ":")
```

Comment:
I wonder if it's better to split this so we have `host`, `port` variables? Might be more readable for future iterations on these tests? 

Could use [net.SplitHostPort](https://golang.org/pkg/net/#SplitHostPort) for this.